.github/issue_template.md

```diff
@@ -59,7 +59,7 @@ ACME Corporation is one of the most...
 
 ## How to apply
 
-Please send an email to [email@email.com](email@email.com) with your resume, use 'Node Developer' as subject
+Please send an email to [email@email.com](mailto:email@email.com) with your resume, use 'Node Developer' as subject
```

Comment:
```suggestion
Please send an email to [email@email.com](mailto:email@example.com) with your resume, use 'Node Developer' as subject
```

Reply:
I'm not quite sure I understand

Reply:
documentation should always refer to example.com as that is an officially reserved name specifically for use as a placeholder in documentation / content templates. email.com is a real domain owned by someone.